frontend/app/src/components/settings/PremiumSettings.vue

```diff
@@ -22,6 +22,7 @@
               prepend-icon="fa-key"
               :disabled="premium && !edit"
               :type="showKey ? 'text' : 'password'"
+              :error-messages="errorMessages"
```

Comment:
I made a new user account to test this. What surprised me is that not only the get premium popup was missing. But the first time I logged in also the "Get Premium" at the top menu was missing.

![2020-08-09_09-14](https://user-images.githubusercontent.com/1658405/89726958-e096a880-da20-11ea-84e1-5a76d19a00e7.png)

Second time it was there fine.

There was this issue where we wanted to remove the popup but not also the menu entry. That should always be there if the user does not have premium.

Reply:
Ok give me some time to verify.

Reply:
It's fixed.